Makefile

```diff
@@ -81,7 +81,7 @@ scorecard: generate go-generate bundle
 
 # Build manager binary
 manager: generate
-	go build -o bin/manager main.go
+	go build -o bin/manager main.go -tags openshift
```

Comment:
Appending the tag `openshift` should be an optional Makefile variable. By default we should not add the flag but Makefile users should be able to do something like `make ... DISTRIBUTION=openshift`